rocketpool/node/collectors/node-collector.go

```diff
@@ -292,6 +294,18 @@ func (collector *NodeCollector) Collect(channel chan<- prometheus.Metric) {
 			return err
 		}
 
+		// Get the totalNodeWeight for the last completed interval
+		previousRewardIndex := state.NetworkDetails.RewardIndex - 1
+		previousInterval, err := rprewards.GetIntervalInfo(collector.rp, collector.cfg, collector.nodeAddress, previousRewardIndex, nil)
+		if err != nil {
+			return err
+		}
+
+		if !previousInterval.TreeFileExists {
+			return fmt.Errorf("Error retrieving previous interval's total node weight: rewards file %s doesn't exist for interval %d", previousInterval.TreeFilePath, previousRewardIndex)
```

Comment:
i think we should log an error here without returning it, otherwise all the node metrics on the grafana dash will die until the tree is downloaded.

Reply:
There is already existing logic immediately below this which iterates through the claimed/unclaimed reward indexes.  Is it possible that for an unclaimed interval (say, immediately after the reward period) that that code will also have the same problem?

Reply:
cool- if it's not a new regression, consider it out of scope